src/file.cr

```diff
@@ -172,6 +172,15 @@ class File < IO::FileDescriptor
     LibC.access(filename, LibC::F_OK) == 0
   end
 
+  # Returns true if file exists and is a file
```

Comment:
Mmh, rereading this, the parameter is called "path", how about "Returns true if the given \*path\* exists and is a file."